server/consumer.go

```diff
@@ -69,6 +70,11 @@ type ConsumerConfig struct {
 	Direct bool `json:"direct,omitempty"`
 }
 
+type PushActive struct {
+	Subject string `json:"subject"`
+	Queue   string `json:"queue,omitempty"`
```

Comment:
but there could be multiple queue groups on the same subject - and some unqueued?

Reply:
No, once a consumer is bound to a queue group, meaning we know you want to do scale out on this consumer, its bound to that queue group.

Reply:
No, 2 nats clients can have queue groups on this subject, 2 different queue groups. And there could also be subscribes on it that has no queue group - while others does.

You specifically take only the first, but thats just not deterministic at all because if there are 2 clients each with its own queue group one server will report them one way and another another way depending on how and when splits happened.

Reply:
Just like two apps having different delivery subjects (e.g. _INBOX.>) only one is bound to the server side consumer. This is similar in that only one queue group should be bound to the delivery subject for scale out binding to a single consumer.

If someone binds that shouldn't, the app that *should* be bound will fail to bind with a queue mismatch error.